src/firecracker/src/main.rs

```diff
@@ -138,11 +138,9 @@ fn main() {
         }
         _ => {
             if let Some(help) = arg_parser.arguments().value_as_bool("help") {
-                if help {
```

Comment:
The default value of `help` is false, so we do need to check that help was configured (meaning that it is some), and that its value is true.

Reply:
yes Andreea, the default value of arg parser which takes no value is `false` but `--help` is handled specially in this current model. So based on the current model only two values are possible either `None` or `Some(true)` for `--help`.

If we are planning to alter the logic of arg parser in the future then yes as you say it is better to leave that `if condition` to support future implementation. I am ok with both the flow 🙂

Reply:
@andreeaflorescu done 👍